src/entry/__tests__/__snapshots__/Entry.spec.tsx.snap

```diff
@@ -357,6 +357,15 @@ exports[`Entry should render main 1`] = `
         <section
           class="base-layout__content"
         >
+          <div
```

Comment:
Sorry that might not have been clear: I think you could just have it as another link next to "add a publication" instead of having it right aligned in a different section.

Reply:
Done